Tools/autotest/param_metadata/rstemit.py

```diff
@@ -241,15 +241,19 @@ def emit(self, g):
 
             if d.get('User', None) == 'Advanced':
                 ret += '\n| *Note: This parameter is for advanced users*'
+            if d.get('RebootRequired', None) == 'True':
```

Comment:
We should validate this if we're going to do strict equivalence here.  Accept only True or False.

We should probably just remove the `False` entries.

Reply:
Removed the `@RebootRequired: False` metadata lines. By validate do you mean enforce case? I grepped for "true" or other odd formats and didn't find anything.

Reply:
> Removed the `@RebootRequired: False` metadata lines. By validate do you mean enforce case? I grepped for "true" or other odd formats and didn't find anything.

Make sure that if it exists it is `True` and only `True`.

Reply:
Fixed and already caught one in CI.